tools/cabana/SConscript

```diff
@@ -16,7 +16,7 @@ qt_libs = ['qt_util'] + base_libs
 
 cabana_env = qt_env.Clone()
 cabana_libs = [widgets, cereal, messaging, visionipc, replay_lib, 'panda', 'avutil', 'avcodec', 'avformat', 'bz2', 'curl', 'yuv', 'usb-1.0'] + qt_libs
-opendbc_path = '-DOPENDBC_FILE_PATH=\'"%s"\'' % (cabana_env.Dir("../../opendbc").abspath)
+opendbc_path = '-DOPENDBC_FILE_PATH=\'"%s"\'' % (cabana_env.Dir("../../opendbc").relpath)
```

Comment:
this breaks cabana when launching not in the root directory. can we have it relative to the binary instead perhaps?

Reply:
![image](https://github.com/commaai/openpilot/assets/9648890/05db0960-a68c-4a50-9799-7b1c9b2cfd9b)

Reply:
(would probably require changes to cabana)

Reply:
Yeah, the macro define during compilation is tricky without using absolute pathing. This is the same issue that I face with  this[ opendbc PR ](https://github.com/commaai/opendbc/pull/990)

Reply:
[get_opendc_path](https://github.com/commaai/openpilot/pull/30857/commits/9fb19fb1a496f97cfa2d03e20caad723b622ec0e). This hack don't make you rely on the macro define anymore

Reply:
@deanlee have any thoughts on this or a better solution? we want to eliminate any abspath as that causes lots of caching issues